tests/test_amity.py

```diff
@@ -0,0 +1,227 @@
+import unittest
+from os import sys, path
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from src.amity import Amity
+from src.room import Room
+from src.fellow import Fellow
+from src.staff import Staff
+from src.office import Office
+from src.livingspace import LivingSpace
+from src.migration import Migration
+
+class TestAmity(unittest.TestCase):
+	"""Test cases for Amity"""
+
+	def add_person(self, living_space=True, reset=True):
+		amity = Amity('add_person')
+		args = {
+		'<firstname>':'test',
+		'<lastname>':'user',
+		'<person_type>':'fellow',
+		'-w':living_space,
+		}
+		if reset:
+			amity.people = []  # make sure the people list is empty
+		amity.run_command(args)
+		return amity
+
+
+	def create_room(self):
+		amity = Amity('create_room')
+		args = {
+		'<room_name>':['testRoom 1','testRoom 2','testRoom 3','testRoom 1'],
+		'<room_type>':['living','office','living'	],
+		}
+		amity.rooms = []  # make sure the rooms list is empty
+		amity.run_command(args)
+		return amity
+
+	def test_0_amity_init_sets_command(self):
```

Comment:
**Testing:** There is no need numbering the tests. What if you were to insert a new test in between others, do you go and change all the subsequent tests?